src/sql-parser/tests/testdata/explain

```diff
@@ -1,232 +1,315 @@
 # Copyright 2020 sqlparser-rs contributors. All rights reserved.
+
 # Copyright Materialize, Inc. and contributors. All rights reserved.
+
 #
+
 # This file is derived from the sqlparser-rs project, available at
+
 # https://github.com/andygrove/sqlparser-rs. It was incorporated
+
 # directly into Materialize on December 21, 2019.
+
 #
+
 # Licensed under the Apache License, Version 2.0 (the "License");
+
 # you may not use this file except in compliance with the License.
+
 # You may obtain a copy of the License in the LICENSE file at the
+
 # root of this repository, or online at
+
 #
-#     http://www.apache.org/licenses/LICENSE-2.0
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
 #
+
 # Unless required by applicable law or agreed to in writing, software
+
 # distributed under the License is distributed on an "AS IS" BASIS,
+
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
 # See the License for the specific language governing permissions and
+
```

Comment:
Something very weird happened to this file. Can we restore its original formatting?

Reply:
My VSCode thinks that this is Markdown and does auto-format the file if I'm not careful.